Client/Frontend/Settings/Clearables.swift

```diff
@@ -33,9 +33,6 @@ class HistoryClearable : Clearable {
             self.profile.thumbnails.clear({ success in
                 SDImageCache.sharedImageCache().clearDisk()
                 SDImageCache.sharedImageCache().clearMemory()
-                self.profile.favicons.clearFavicons().upon {
```

Comment:
I don't think we want to do that, do we? That would erase even the bookmarks favicons. Clearing the history should cascade to delete the favicons we want...